libafl_frida/src/asan/asan_rt.rs

```diff
@@ -2648,7 +2647,7 @@ impl AsanRuntime {
                 }
                 X86Register::Rdi => {
                     // In this case rdi is already clobbered, so we want it from the stack (we pushed rdi onto stack before!)
-                    writer.put_mov_reg_reg_offset_ptr(X86Register::Rsi, X86Register::Rsp, -0x28);
+                    writer.put_mov_reg_reg_offset_ptr(X86Register::Rsi, X86Register::Rsp, -0x18);
```

Comment:
hm.. why? `rdi` is on `stack-0x28`

Reply:
https://github.com/AFLplusplus/LibAFL/pull/1314